scripts/py_matter_idl/matter_idl/matter_grammar.lark

```diff
@@ -78,13 +87,13 @@ bool_default: "true"i  -> bool_default_true
             | "false"i -> bool_default_false
 ?default_value: "default"i "=" (integer | ESCAPED_STRING | bool_default)
 
-constant_entry: id "=" positive_integer ";"
+constant_entry: [maturity] id "=" positive_integer ";"
 positive_integer: POSITIVE_INTEGER | HEX_INTEGER
 negative_integer: "-" positive_integer
 
 integer: positive_integer | negative_integer
 
-struct_field: member_attribute* field
+struct_field: [maturity] member_attribute* field
```

Comment:
What about command fields?  Event fields?  Or do they all use this production?

Reply:
structs apply to all commands (they are structs for both in and out) as well as events (they use struct field too).